sqliterepo/repository.c

```diff
@@ -359,6 +360,46 @@ sqlx_repository_init(const gchar *vol, const struct sqlx_repo_config_s *cfg,
 	return NULL;
 }
 
+void
+sqlx_repository_initial_cleanup(sqlx_repository_t *repo)
+{
+	gchar pattern[PATH_MAX] = {0};
+	glob_t buf = {0};
+	int rc;
+
+	g_assert_nonnull(repo);
+
+	g_snprintf(pattern, sizeof(pattern), "%s/tmp/dump.sqlite3.*",
```

Comment:
We should also check for `%s/tmp/restore.sqlite3.*`. See https://github.com/open-io/oio-sds/blob/4.2.x/sqliterepo/repository.c#L1584.